paper.md

```diff
@@ -42,65 +42,69 @@ Born of idea to provide a unified data distribution for neuroscience, taking a v
-remotes to facilitate access to an ever growing collection of resources [@git-annex:special_remotes] and to overcome technological limitations (e.g., maximal file sizes, or file system inodes limits).
+Git can interact with other repositories on the file system or accessible via a set of standard (ssh, http) or custom (git) network transport protocols.
+Interaction with non git-aware portals should then be implemented via custom git transfer protocols, as it e.g. was done in git-remote-rclone [TODOREF].
+Git-annex provides access to a wide range of external data storage resources via various protocols, but cannot implement all idiosyncracies of any individual data portal.
+In particular, scientific data is frequently stored in compressed archives to reduce its storage demands, or on specialized servers, such as XNAT (www.xnat.org).
+To address these demands, git-annex implemented support for external special remotes [@git-annex:special_remotes_protocol].
+This allowed DataLad and many other projects to access and provide git-annex special remotes to facilitate access to an ever growing collection of resources [@git-annex:special_remotes] and to overcome technological limitations (e.g., maximal file sizes, or file system inodes limits).
 
 **They require a layer above to establish a *distribution*.**
-DataLad project's initial goal was to provide a data distribution with unified access to the already available public data archives in neuroscience, such as http://crcns.org, http://openfmri.org, etc. 
-git and git-annex are just "client tools" and on their own do not provide aggregation of different resources into a unified distribution.
-http://datasets.datalad.org became an example of such a data distribution curated by DataLad team, which at the moment provides access to over 250 TBs of data across wide range of projects and archives.  
+The DataLad project's initial goal was to provide a data distribution with unified access to the already available public data archives in neuroscience, such as http://crcns.org or http://openfmri.org.
+Git and git-annex are just "client tools" and do not provide aggregation of different resources into a unified distribution on their own.
+http://datasets.datalad.org became an example of such a data distribution curated by the DataLad team, which at the moment provides streamlined access to over 250 TBs of data across wide range of projects and archives.
 
 **Modularization is needed to scale.**
 Research workflows impose additional demands for an efficient research data management (RDM) platform besides "version control" and "data transport".
-Many research datasets contain millions of files, and that precludes placing such datasets in their entirety within a single git repository even if files are tiny in their size.
+Many research datasets contain millions of files, and that precludes placing such datasets in their entirety within a single git repository even if individual files are tiny in their size.
```

Comment:
Not sure, where to put this comment, so abusing your PR ;-)

I think there's another aspect of modularization, that we didn't mention yet, but relates closely to the previously pointed to notion of "like software distribution". That is the notion of dependencies. Not sure whether to point that out in this paragraph or rather early on, when we mention the original idea of "data packages like software packages".

Reply:
Better to put a comment in the manuscript. It will get lost here.